### DISCUSSION TITLE
Vehicle Simulator

### DISCUSSION THREAD

**Discussion by saariniemi** [VDA5050/VDA5050#127] [Q&A]
I’m trying to get in contact with anyone who can provide AGV Simulator for VDA 5050 vehicles.

Anyone who can help me?

**Comment by jefersonflamini** [comment]
Hi @saariniemi.
What features do you need to have in this simulator?
Is the purpose to test compliance with the specification or to test your AGV management application?
What type of AGV (free navigation, central map, guided vehicles)?
